Permanent/Modules/Onboarding/Views/OnboardingWelcomeView.swift

```diff
@@ -11,35 +11,75 @@ struct OnboardingWelcomeView: View {
     var buttonAction: (() -> Void)
     
     var body: some View {
-        ZStack(alignment: .bottom) {
-            VStack(alignment: .leading, spacing: 24) {
+        if Constants.Design.isPhone {
+            ZStack(alignment: .bottom) {
+                VStack(alignment: .leading, spacing: 24) {
+                    VStack(alignment: .leading) {
+                        HStack(spacing: 0) {
+                            Text("Hello, ")
+                                .textStyle(UsualXLargeLightTextStyle())
+                                .foregroundColor(.white)
+                            Text("\(fullName)")
+                                .textStyle(UsualXLargeBoldTextStyle())
+                                .foregroundColor(.white)
+                            Text("!")
+                                .textStyle(UsualXLargeLightTextStyle())
+                                .foregroundColor(.white)
+                            Spacer()
+                        }
+                        Text("Welcome to")
+                            .textStyle(UsualXLargeLightTextStyle())
+                            .foregroundColor(.white)
+                        Text("Permanent!")
+                            .textStyle(UsualXLargeLightTextStyle())
+                            .foregroundColor(.white)
+                    }
+                    Text("We’re so glad you’re here!\n\nAt Permanent, it is our mission to provide a safe and secure place to store, preserve, and share the digital legacy of all people, whether that's for you or for your friends, family, interests or organizations.\n\nWe know that starting this journey can sometimes be overwhelming, but don’t worry. We’re here to help you every step of the way.")
```

Comment:
as this text is repeating itself, maybe create a variable for it instead of copy pasting it.